test/match.js

```diff
@@ -8,12 +8,12 @@ describe('match', function() {
 
   it('determines whether a string matches a regex', function() {
     assert.strictEqual(R.match(re, 'B17-afn').length, 1);
-    assert.strictEqual(R.match(re, 'B1-afn'), null);
+    assert.deepEqual(R.match(re, 'B1-afn'), []);
```

Comment:
We could switch assertion libraries. Using `assert.deepEqual` is almost always a bad idea. ;)